engine/table/src/test/java/io/deephaven/engine/table/impl/select/WhereFilterTest.java

```diff
@@ -16,60 +18,194 @@ public class WhereFilterTest extends TestCase {
 
     private static final ColumnName FOO = ColumnName.of("Foo");
     private static final ColumnName BAR = ColumnName.of("Bar");
-    private static final Value V42 = Value.of(42L);
+    private static final ColumnName BAZ = ColumnName.of("Baz");
+    private static final Literal V42 = Literal.of(42L);
 
-    public void testIsNull() {
+    public void testFoo() {
+        expect(FOO, ConditionFilter.class, "Foo");
+    }
+
+    public void testNotFoo() {
+        opposite(FOO, ConditionFilter.class, "!Foo");
+    }
+
+    public void testFooIsTrue() {
+        expect(Filter.isTrue(FOO), MatchFilter.class, "Foo in [true]");
+    }
+
+    public void testFooIsFalse() {
+        expect(Filter.isFalse(FOO), MatchFilter.class, "Foo in [false]");
+    }
+
+    public void testFooIsNotTrue() {
+        // This is *not* logically equivalent to "Foo in [false]"
```

Comment:
Because you felt this was worth a comment, I suggest you finish out the comment with a clause or second sentence explaining why it's "*not* logically equivalent" so the reader doesn't have to pause to puzzle it out.